lib/note-info.jsx

```diff
@@ -30,7 +42,7 @@ export default React.createClass( {
 		const isPinned = includes( data.systemTags, 'pinned' );
 		const isMarkdown = includes( data.systemTags, 'markdown' );
 		const isPublished = includes( data.systemTags, 'published' );
-		const publishURL = isPublished && data.publishURL !== '' && data.publishURL && `http://simp.ly/publish/${data.publishURL}` || null;
+		const publishURL = isPublished && data.publishURL !== '' && data.publishURL && `http://simp.ly/p/${data.publishURL}` || null;
```

Comment:
Similar place as above to simplify. This just seems like a crazy line to parse and understand what the code's intent was. I keep thinking, "am I missing something? why is this such a long construct?"